src/equations/2d/compressible_euler.jl

```diff
@@ -4,7 +4,7 @@
 
 The compressible Euler equations for an ideal gas in two space dimensions.
 """
-struct CompressibleEulerEquations2D{RealT} <: AbstractCompressibleEulerEquations{2, 4}
+struct CompressibleEulerEquations2D{RealT<:Real} <: AbstractCompressibleEulerEquations{2, 4}
```

Comment:
Why this change? Just as a precaution that nobody tries to do this for `RealT === Vector{Float64}`?

Reply:
Correct.

Reply:
:+1: